fragdenstaat_de/fds_cms/templates/foirequest/cms_plugins/map.html

```diff
@@ -0,0 +1,311 @@
+{% load i18n %}
+{% load static %}
+{% load leaflet_tags %}
+{% leaflet_js %}
+{% leaflet_css %}
+<div class="foirequest-map-container">
+    <div id="map" style="height: 600px;"></div>
```

Comment:
This currently only supports one map element per page. This should use the plugin ID to make it unique.